paper/paper.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'MOTrainer: Distributed Measurement Operator Trainer for Data Assimilation Applications'
+tags:
+  - Python
+  - Measurement Operator
+  - Data Assimilation
+  - Machine Learning
+  - Kalman Filter
+authors:
+  - name: Ou Ku
+    orcid: 0000-0002-5373-5209
+    affiliation: 1 
+  - name: Fakhereh Alidoost
+    affiliation: 1
+  - name: Xu Shan
+    affiliation: 2
+  - name: Pranav Chandramouli
+    affiliation: 1
+  - name: Sonja Georgievska
+    affiliation: 1
+  - name: Meiert W. Grootes
+    affiliation: 1 
+  - name: Susan Steele-Dunne
+    corresponding: true
+    affiliation: 2
+affiliations:
+ - name: Netherlands eScience Center, Netherlands
+   index: 1
+ - name: Delft University of Technology, Netherlands
+   index: 2
+date: 22 Dec 2023
+bibliography: paper.bib
+---
+
+## Summary
+
+Data assimilation is an essential tool in Earth and environmental sciences, enabling physical model states to be constrained using observational data.
+
+In the DA process, observations are integrated into the physical model through the application of a Measurement Operator (MO) – a connection model mapping physical model states to observations. Researchers have observed that employing a Machine-Learning (ML) model as a surrogate MO can bypass the limitations associated with using an overly simplified MO [@Forman:2014; @XUE:2015; @Forman:2017].
+
+## Statement of Need
+
+A surrogate MO, as a ML model, should be trained with the assumption that a single MO applies when mapping physical model states to observations. When dealing with a large spatio-temporal scale, multiple mapping processes may exist, prompting consideration for training separate MOs for distinct spatial and/or temporal partitions of the dataset. As the number of partitions increases, a challenge arises in distributing these training tasks effectively among the partitions.
```

Comment:
```suggestion
A surrogate MO, as a ML model, is trained with the assumption that a single MO applies when mapping physical model states to observations. When dealing with a large spatio-temporal scale, multiple mapping processes may exist, prompting consideration for training separate MOs for distinct spatial and/or temporal partitions of the dataset. As the number of partitions increases, a challenge arises in distributing these training tasks effectively among the partitions.
```

Reply:
@rogerkuou thanks for the suggestion! Sorry I updated you change by further deleting that comma.